### PR TITLE
Backend changes for frontend gotowanie

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: src/main/java/pl/edu/pw/mwotest/DataLoader.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    @Autowired
+    public DataLoader(ClientSeeder clientSeeder, ProductSeeder productSeeder, OrderSeeder orderSeeder) {
+        this.clientSeeder = clientSeeder;
+        this.productSeeder = productSeeder;
+        this.orderSeeder = orderSeeder;
+    }
+
+    @Override
+    public void run(String... args) {
```
Comment (ochnios) [inline src/main/java/pl/edu/pw/mwotest/DataLoader.java:24]: Dałoby radę jakoś wyłączyć seedery dla testów żeby nam się nie wywalały? Bo chyba część przez to nie przechodzi. Jak nie to ez.

File: pom.xml [hunk 2]
```diff
@@ -33,7 +38,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-
+        <dependency>
```
Comment (ochnios) [inline pom.xml:41]: Ta zależność chyba jest zbędna.

File: pom.xml [hunk 3]
```diff
@@ -17,6 +17,11 @@
         <java.version>17</java.version>
     </properties>
     <dependencies>
+        <dependency>
```
Comment (ochnios) [inline pom.xml:20]: Tej libki 3 lata już nie aktualizowali i ma podatności... ale w tym projekcie to w sumie bez znaczenia

File: src/main/java/pl/edu/pw/mwotest/DataLoader.java (hunk 1 above)
Comment (TPGPL) [reply src/main/java/pl/edu/pw/mwotest/DataLoader.java:24]: no mozna zakomentowac te  wywolania seed() i bedzie git, ale to i tak jest bardziej branch do prezentacji fronta, a nie zeby zmergowac

Comment (ochnios) [reply src/main/java/pl/edu/pw/mwotest/DataLoader.java:24]: aa no to luz
